nodejs/week3/warmup/index.js

```diff
@@ -0,0 +1,45 @@
+const express = require("express");
+const app = express();
+
+app.use(express.json()); // for parsing application/json
+app.use(express.urlencoded({ extended: false })) // for parsing application/x-www-form-urlencoded
+
+// Calculator using :param
+app.get('/calculator/multiply', (req, res) => {
```

Comment:
The idea with this exercise was to have the part after `/calculator/` to be a param, like this:

```suggestion
app.get('/calculator/:method', (req, res) => {
```

And then depending on that parameter the calculator would do different operations.